### ISSUE TITLE
Can't convert a link in a table in markup language.

### ISSUE THREAD

**Issue by lyenliang** [catcombo/jira2markdown#22]
Hi, I found that I can't convert a link in a table using `jira2markdown.convert()`. 

Here's my code:

```
import jira2markdown

markup = '||aa||bb||cc||dd||\n|row1|row1-1|row1-2|row1-3[mylink|https://www.google.com]|'
result = jira2markdown.convert(markup)
print(result) # shows '|aa|bb|cc|dd|\n|-|-|-|-|\n|row1|row1-1|row1-2|row1-3|\n'
```

As you can see from the result above, the link `[mylink|https://www.google.com]` disappears after the conversion.

I use python `3.8.17` in macOS Monterey 12.6 to run the above code. 

And I use the latest version(`0.3.4`) of jira2markdown.

This error doesn't occur when I run it in macOS Ventura 13.4.1. 

How can I run `jira2markdown` correctly in macOS Monterey 12.6?

What are the dependent libraries of `jira2markdown`?

**Comment by lyenliang** [comment]
Thank you for your information. I just downgraded my `pyparsing` from `3.1.0` to `3.0.9`. And this solves my problem.

**Comment by catcombo** [comment]
It looks like a bug in the `SkipTo` operator. I've opened an issue in the `pyparsing` repository https://github.com/pyparsing/pyparsing/issues/500 I also released version 0.3.5 to temporarily fix this problem by pining the `pyparsing` version to 3.0.9.

**Comment by catcombo** [comment]
The bug was fixed by `pyparsing` team. I released a version 0.3.6 to unpin `pyparsing` version in dependencies. @lyenliang  thank you once again for the issue!
